src/OrbitGl/TrackTestData.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "TrackTestData.h"
+
+namespace orbit_gl {
+
+// This is copied from CallTreeViewItemModelTest.cpp, we may want to supply this in a header,
+// but it is not clear what we want to be testing here in the future.
```

Comment:
I don't understand this. This is in a header already with this change, so why copy it instead of extracting it to a single place? Especially since you want to use it in `OrbitQt` as well.

Reply:
I actually simply removed the comment for now because I think it's more confusing than helpful. I copied the data because I think it's reasonable to have separate test data for the track part of OrbitGl, I ended up doing minor additions and did not want to mess with the test data of other Qt models.

This data could be extended later to cover other tests around tracks, so it's mainly intended for OrbitGl. The usage in OrbitQt is simply required because I'm testing the model that relies on the created tracks.

Reply:
I was about to ask, thanks for the clarification.